src/confd/yang/infix-if-vlan@2023-10-25.yang

```diff
@@ -0,0 +1,65 @@
+submodule infix-if-vlan {
+  yang-version 1.1;
+  belongs-to infix-interfaces {
+    prefix infix-if;
+  }
+
+  import ietf-interfaces {
+    prefix if;
+    reference
+      "RFC 8343: A YANG Data Model For Interface Management";
```

Comment:
References are not really needed in our native Infix models, just for future reference.  Also, I've tried to keep oneliner values on the same line as the key.  Nothing major.